src/types/help-roles.ts

```diff
@@ -0,0 +1,11 @@
+const languages = ["Javascript", "Typescript", "Python", "Rust", "Java", "Kotlin", "CSharp", "CPlusPlus", "C", "Go", "Assembly", "PHP", "Lua", "Swift", "Haskell", "Dart", "Ruby", "Elixir", "Lisp", "Perl", "Scala", "Clojure", "Other"]
+
+let platforms = ["Discordjs", "Discordpy", "HTMLCSS", "MongoDB", "SQL", "React", "Other"]
+
+let helpRoles = [...languages, ...platforms]
+
+export {
```

Comment:
These aren't types, so these would be better fit for somewhere like utilities.

Reply:
Fixed in https://github.com/tm21cy/Bit/pull/13/commits/7ca3cf6d4cb364bbc50b3996f2f69a9434a96702.